docs/overview_and_guide.rst

```diff
@@ -227,20 +237,84 @@ After account creation, launch Raiden with the path of your keystore supplied::
+
+where ``network`` is one of ``mainnet``, ``ropsten``, ``rinkeby`` or ``goerli``.
+
+Log into MyEtherWallet, the geth console or any other Ethereum interface you like with an account
+that holds enough Raiden tokens, and call ``UserDeposit.deposit(<beneficiary>, <total_deposit>)``.
+
+Here ``total_deposit`` is the amount of Raiden tokens you wish to deposit and ``beneficiary`` is the
+Ethereum address of your Raiden node.
+
+For example, suppose we use Raiden with the address ``0x3040435D7F1012e861f0B0989422a47D1825F120`` on the mainnet and want
+to deposit 0.001 Raiden tokens (10**15 REI) for our Raiden node to use. We take a look at ``deployment_services_mainnet.json``, which yields:
+
+.. code-block:: none
+
+    {
+        "contracts_version": null, "chain_id": 1,
+        (...)
+        "UserDeposit": {
+            "address": "0x53Cc1decDD7d452c8844a5f383e23AD479A1f614",
+            (...)
 
-Raiden provides a pathfinding service for efficient transfer routing. The default option when starting the client is with the pathfinding service to be paid in RDN tokens.
+
+Using MyEtherWallet, we log into our account that holds the Raiden tokens (which must not be the same that we use
+for the Raiden node, since that is supposed to be used only by Raiden) and look up the ``UserDeposit`` contract at
+``0x53Cc1decDD7d452c8844a5f383e23AD479A1f614``, the address we found in the deployment JSON file.
+
+In the contract interface on MyEtherWallet, we set ``method`` to ``deposit``, ``beneficiary`` to our Raiden address
+``0x3040435D7F1012e861f0B0989422a47D1825F120`` and ``total_deposit`` to ``1000000000000000`` and send the transaction.
+
+Optional CLI arguments
+======================
+
+There are further CLI arguments with which you can control, among other things
+
+ 1. The choice of a pathfinding service
+ 2. The choice of a monitoring service
+ 3. Logging
+
+1. Pathfinding service
+**********************
+
+A pathfinding service is a third party service helping your node with efficient transfer routing. It is usually paid in RDN tokens.
+
+Raiden can be configured to not use a pathfinding service and rely on its internal routing instead.
+This is discouraged since the node has less information about the network that it can use to find routes and compute fees.
+So transfers will be more likely to fail to route, and paid fees will often be unnecessarily high when internal routing is used.
+If you want to use internal routing anyway, you can do so with ``--routing-mode local``.
+
+.. note::
+    Although otherwise discouraged, ``--routing-mode local`` has to be used at the moment to try
+    out Raiden on the mainnet. The Raiden Service Bundle, with pathfinding service and registry,
+    will not be deployed on the mainnet until the Alderaan release.
+
+If you want to use a particular pathfinding service, e. g. one of the testnet pathfinding services given below, you can
```

Comment:
There is one space too much between `e.` and `g.`, right?

Reply:
Really? I wasn't sure if its spelled with a whitespace